packages/qdrant-loader-mcp-server/src/qdrant_loader_mcp_server/search/hybrid/pipeline.py

```
@@ -14,6 +15,24 @@
+    Tolerant of both sync and async implementations and of searchers that don't
+    expose the hook at all (legacy / test doubles).
+    """
+    probe = getattr(vector_searcher, "supports_qdrant_hybrid", None)
+    if not callable(probe):
+        return False
+    try:
+        result = probe()
+        if inspect.isawaitable(result):
+            result = await result
+        return bool(result)
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)